lib/snowpack/query.ex

```diff
@@ -35,9 +35,7 @@ defmodule Snowpack.Query do
             opts :: Keyword.t()
           ) :: [Type.param()]
     def encode(_query, params, opts) do
-      params
-      |> List.flatten()
```

Comment:
Don't want to flatten a param of type List. Not that List params are supported yet, but this just makes no sense.

`[1, 2, ["a", "b"], 5]` --> `[1, 2, "a", "b", 5]` is not what we want.